rules/or/or-OdiScript.js

```diff
@@ -12,32 +12,32 @@
 		version: '1.0',
 		contextLength: 6,
 		maxKeyLength: 4,
-		patterns: [
+			patterns: [
```

Comment:
)$ jscs --preset=wikimedia  rules/or/or-OdiScript.js 
Expected indentation of 2 characters at rules/or/or-OdiScript.js :
    13 |  contextLength: 6,
    14 |  maxKeyLength: 4,
    15 |   patterns: [
----------^
    16 |   ['/([A-Za-z\\>_\\-\\!@#\\$\\^\\&\\%\\*\\~\\.\\|\\\\0-9])', '$1'],
    17 |   ['\'([\'])','\"'], // '' to "